source/flux/parts/stoprint.js

```diff
@@ -9,6 +9,9 @@ import {
 	fetchRecentPrinters,
 	logIn,
 } from '../../lib/stoprint'
+import {API} from '../../globals'
+import * as defaultData from '../../../docs/color-printers.json'
```

Comment:
I don't think we should bundle this list with the app. Unlike the other resources, it's not standalone information, and you need a network connection already to be able to see the print jobs.